reference-architecture/vmware-ansible/roles/haproxy-server/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Ensure HAProxy is installed.
+  yum: name=haproxy state=installed
+  when: inventory_hostname == 'haproxy-0'
```

Comment:
The conditionals on inventory_hostname seem odd here. Instead, I would expect that the playbook that applies this role to hosts would only apply it to the proper hosts.